### ISSUE TITLE
Release 1.1.0

### ISSUE THREAD

**Issue by benoit74** [openzim/kolibri#45]
- [x] Check that dependencies have been updated to latest version (especially python-scraper lib)
- [x] Adjust version in `__about__.py` to `x.x.x`
- [x] Update Github milestone to match the issues that will be released 
- [x] Close Github milestone
- [x] Update the Changelog so that it is in line with the content of Github milestone
- [x] Push a tag on Github named `vx.x.x`
- [x] Create a Github release, pointing to the tag, with the Changelog of this release
- [x] Publish the Github release (this will trigger the CI, if the CI fails and you have to push a minor fix which does not justify to create a new version, you will have to delete the release and re-create it from scratch)
- [x] Check that version is published as a Github release at https://github.com/openzim/kolibri/releases
- [x] Check that version is published on Github Container Registry at https://ghcr.io/openzim/kolibri and tagged `latest`
- [x] Check that version is published on Pypi at https://pypi.org/project/kolibri2zim/
- [x] Create a new Github milestone with the next minor version incrementaly
- [x] Create a new Github issue attached to this milestone with this checklist inside
- [x] Create new `## [Unreleased]` section in Changelog (placeholder for future entries)
- [x] Adjust version in `__about__.py` to `x.y.z-dev0
- [x] Inform rgaudin that a new release is ready to use so that he will update Zimfarm recipes
- [x] If needed, open a PR on Zimfarm to add support for new CLI parameters of interest

**Comment by kelson42** [comment]
@benoit74 Would be greate to put all issue tickets solved since last release in the milestone `1.1.0`. If a PR has been made directly, so without a corresponding issue, then please put the PR itself in the milestone (other just leave the "milestone"property alone to avoid duplicates with issues).

**Comment by benoit74** [comment]
@kelson42 it is already done, do you have something specific in mind which is missing and that might have slipped my checks ?
Btw, I added all old issues to a "virtual" v0 milestone, so that now you can check that all closed issues have a milestone assigned or are marked as `duplicate` or `wontfix`

**Comment by benoit74** [comment]
Release completed, update to zimfarm recipes done myself, PR pending on Zimfarm for new CLI parameter: https://github.com/openzim/zimfarm/issues/805
